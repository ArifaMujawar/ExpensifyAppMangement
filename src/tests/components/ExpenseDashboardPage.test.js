import React from 'react';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';

test('should render Expense Dashboard Page correctly',()=>{
    const wrapper = shallow(< ExpenseDashboardPage/>);
    expect(toJSON(wrapper)).toMatchSnapshot();
});