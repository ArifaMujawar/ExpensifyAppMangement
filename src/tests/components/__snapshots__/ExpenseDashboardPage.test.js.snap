// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense Dashboard Page correctly 1`] = `
<div>
  <Connect(ExpenseList) />
  <Connect(ExpenseListFilters) />
</div>
`;
